.github/workflows/pull-requests.yml

```diff
@@ -69,6 +69,12 @@ jobs:
       - name: Build & Test
         run: pnpm build --concurrency=100%
 
+      - name: Download Test Validator Log
+        uses: actions/download-artifact@v2
+        with:
+          name: validator-log
+          path: test-ledger/validator.log
+
```

Comment:
👀

Reply:
Haha. Harsh.
<img width="345" alt="image" src="https://github.com/solana-labs/solana-web3.js/assets/13243/e80112f2-40a8-4222-9b89-1eeb973a6d70">

Reply:
Lmao I just saw. I'll have to update it to not skip on fail. Should have known. 🫠